docs/content/config/best-practices/dkim_dmarc_spf.md

```diff
@@ -156,6 +156,13 @@ DKIM is currently supported by either OpenDKIM or Rspamd:
         use_esld = true;
         check_pubkey = true; # you want to use this in the beginning
 
+        selector = "mail";
+        # The path location is searched for a DKIM key with these variables:
+        # - `$domain` is sourced from the MIME mail message `From` header
+        # - `$selector` is configured for `mail` (as a default fallback)
+        path = "/tmp/docker-mailserver/dkim/keys/$domain/$selector.private";
```

Comment:
This path is wrong, currently the keys are in: `/tmp/docker-mailserver/opendkim/keys/`

I have used `path = "/etc/opendkim/keys/$domain/$selector.private";` until now

Reply:
> This path is wrong

It's an example and was explicitly changed to that by me for the docs.

The [contributor used `/tmp/docker-mailserver/...`](https://github.com/docker-mailserver/docker-mailserver/pull/3702/files/02f09ef0e0570344248eadbcea39f2b595494762#diff-aee5f16ae135f9232103399f5670afb3c353405767d6020b38e51480c651d83f) instead of `/etc/...` but otherwise they also referred to `opendkim`.

Not sure why you'd want OpenDKIM as the path for rspamd though? I want this to be a unified location for DKIM keys, but someone will need to contribute that change.

Reply:
Well, it's more that rspamd wanted to do signing before I would even know it. So I patched the path and selector to use open dkim path. Since opendkim was the first dkim stuff I had setup
Just feels like there is too many moving components in DMS and without a diagram I can not decide to shut down one component or another.
Anyway this config allows to local dkim sign and that's all I needed to pass 👍